ojbackend/dispatcherFunc.py

```diff
@@ -0,0 +1,76 @@
+"""
+	dispatcherFunc.py
+	This file provides related functions for dispatcher.py.	
+"""
+from bash import bash
+import MySQLdb
+
+
+def getMachine():
```

Comment:
Want to discuss this implement.
How about loading config when dispatcher starts and sync the configuration periodically?
Implement singleton for configuration .